resources/automagic_dashboards/table/TransactionTable.yaml

```diff
@@ -1,87 +1,7 @@
 title: A look at your [[this]]
 transient_title: It looks like your [[this]] has transactions, so here''s a look at them
 description: Some metrics we found about your transactions.
-metrics:
-- AvgDiscount:
-    metric: [/, [sum, [dimension, Discount]], [sum, [dimension, Income]]]
-    name: Average discount %
-- TotalIncome:
-    metric: [sum, [dimension, Income]]
-    name: Total income
-- AvgIncome:
-    metric: [avg, [dimension, Income]]
-    name: Average income per transaction
-- AvgQuantity:
-    metric: [avg, [dimension, Quantity]]
-    name: Average quantity
-- TotalOrders:
-    metric: [count]
-    name: Number of orders
-dimensions:
-- Timestamp:
-    field_type: CreationTimestamp
-    score: 100
-- Timestamp:
-    field_type: CreationDate
-    score: 100
-- Timestamp:
-    field_type: DateTime
-    score: 90
-- Timestamp:
-    field_type: Date
-    score: 90
-- State:
-    field_type: GenericTable.State
-- Country: GenericTable.Country
-- Income: Income
-- Discount: Discount
-- Quantity:
-    field_type: Quantity
-- SourceSmall:
-    field_type: GenericTable.Source
-    max_cardinality: 5
-- SourceMedium:
-    field_type: GenericTable.Source
-    max_cardinality: 10
-    score: 90
-- SourceLarge:
-    field_type: GenericTable.Source
-- Cohort:
-    field_type: UserTable.JoinTimestamp
-    score: 100
-- Cohort:
-    field_type: UserTable.CreationTimestamp
-    score: 90
-- Cohort:
-    field_type: UserTable.DateTime
-    score: 80
-- ProductMedium:
-    field_type: Product
-    score: 90
-    max_cardinality: 10
-- ProductMedium:
-    field_type: ProductTable.Name
-    max_cardinality: 10
-- ProductMedium:
-    field_type: ProductTable.Title
-    max_cardinality: 10
-- ProductLarge:
-    field_type: Product
-    score: 90
-- ProductLarge:
-    field_type: ProductTable.Name
-- ProductLarge:
-    field_type: ProductTable.Title
-- ProductCategoryMedium:
-    field_type: ProductTable.Category
-    named: category
-    max_cardinality: 10
-    score: 90
-- ProductCategoryLarge:
-    field_type: ProductTable.Category
-    named: category
-- Long: GenericTable.Longitude
-- Lat: GenericTable.Latitude
+data_model: Orders
 filters:
```

Comment:
Micro-question -- 

what exactly is going on with this vs segments in the data model file?

eg, should the x-rays actually have any definitions of segments or common filters at all?

Reply:
These are meant more for things like "last 30 days" which are not really semantical breakdown of data, but I could see the argument for moving these to the data model definition as well.